src/Main.elm

```diff
@@ -37,25 +46,17 @@ update msg model =
 
 ---- VIEW ----
 
-
 view : Model -> Html Msg
 view model =
-    Grid.container []
+        div []
             [ CDN.stylesheet
-            , Grid.gridx [] []
-            , p [] []
+            , CDN.stylesheetFont
+            , CDN.stylesheetTheme
+            , grid
+                [ container Fluid, direction X, gutter Margin ]
```

Comment:
@debjitbis08 I designed the grid fn as per your suggestion and it looks way better, however the first argument of the grid function still takes list of string, I would want it to be list of "config"/custom-type(which will restrict the user) and somehow not able to get my head around this.
Still reading around the web :)